css-layout-classes/styles.css

```diff
@@ -0,0 +1,51 @@
+* {
+  font-family: sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-third {
+  width: 100%;
+  flex-wrap: nowrap;
+  flex-basis: calc(100% / 3);
+  text-align: center;
+}
+
+.column-fourth {
+  flex-basis: 25%;
+}
+
+.characters {
+  max-height: 100%;
+  flex-basis: auto;
+  width: 100%;
+}
+
+.par1-2 {
+  display: flex;
+  width: 75%;
+  font-size: 18px;
+}
```

Comment:
`.par1-2` isn't descriptive. Is this supposed to be a row? A column? I would delete this completely. Your class names should follow the structure in the instructions. `.row` or `column` classes